web/src/main/java/config/YggdrasilWebConfig.java

```diff
@@ -27,6 +27,7 @@ public void addResourceHandlers(ResourceHandlerRegistry registry) {
     public InternalResourceViewResolver viewResolver() {
         InternalResourceViewResolver viewResolver = new InternalResourceViewResolver();
         viewResolver.setViewClass(JstlView.class);
+        viewResolver.setContentType("text/html; charset=utf-8");
```

Comment:
I will leave it for this PR, as this will be my next issue to resolve and this line probably hides the solution :)